Makefile

```diff
@@ -44,6 +44,12 @@ cephcsi:
 	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o  _output/cephcsi ./cmd/
 
+rbdplugin: cephcsi
+	cp _output/cephcsi _output/rbdplugin
```

Comment:
we can discard the cp command for both rbd and cephfs.  `_output/rbdplugin` `_output/cephfsplugin` not used

Reply:
We (@Madhu-1 and I) decided that docs talk about these binaries and for 1.0 we retain status quo and hence copy the same.